documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc

```diff
@@ -172,6 +172,7 @@ secure-boot / file-system encryption is not required, then the device private ke
 See https://gitlab.com/cryptsetup/cryptsetup[Cryptsetup] for more information about open-source disk encryption.
 
 ==== Program a key into OTP with `rpi-otp-private-key`
+NOTE: The `rpi-otp-private-key` script can only be used on devices with the Broadcom BC2711 or BC2712 processor.
```

Comment:
You've missed the 'M' out of BCM2711 and BCM2712 :wink: 
https://www.raspberrypi.com/documentation/computers/processors.html

Reply:
Good catch, thank you! I've force-pushed an amended commit rather than adding a typo fix commit, hope that is okay. 🙂